### PR TITLE
docs: fix empty docs landing page

### LINKED ISSUE #176: fix: helia api docs have no blank pages
## Goals

tsdoc generated documentation at https://ipfs.github.io/helia/ should present users with a more intuitive experience.

```[tasklist]
### Tasks
- [x] fix empty pages
- [x] fix landing pages
- [x] warning The --name option was not specified, and no package.json was found. Defaulting project name to "Documentation".
- [ ] Investigate the blank pages after building on CI
- [ ] investigate: https://ipfs.github.io/helia-ipns/
```

### PR DESCRIPTION
**Pull request by SgtPooki**
fixes #176

### CODE REVIEW

File: serve-docs.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+fastify.get('/helia', async (request, reply) => {
+  // serve the contents of the .docs folder
+  return reply.sendFile('index.html')
+})
+
+fastify.listen({ port: 8448 }, (err, address) => {
+  if (err) throw err
+  // Server is now listening on ${address}
+  console.info(`Server is now listening on ${address}`)
+})
```
Comment (SgtPooki) [inline serve-docs.js:45]: this file isn't required, and I had to revert some things to save it from being deleted, but it helped with confirming functionality

File: typedoc.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "./packages/interop",
+  ],
+  "entryPointStrategy": "packages",
+  "out": ".docs",
+  "hideGenerator": true,
+  "name": "Helia",
+  "includeVersion": true,
+  "cleanOutputDir": true,
+  "logLevel": "Verbose",
+  "readme": "./README.md",
```
Comment (SgtPooki) [inline typedoc.json:15]: this is ultimately the fix that made the landing page better.

**Review by SgtPooki — COMMENTED** [review]
self review, but honestly, https://github.com/ipfs/helia/pull/183 is much better

**Comment by SgtPooki** [comment]
pushing to gh-pages on sgtpooki fork via https://github.com/SgtPooki/helia/actions/runs/5548524680

**Comment by SgtPooki** [comment]
Seems fixed at https://sgtpooki.github.io/helia, removing serve-docs file and merging

File: serve-docs.js (hunk 1 above)
Comment (SgtPooki) [inline serve-docs.js:45]:
```suggestion
```

**Review by BigLep — COMMENTED** [review]
I'll leave to other to approve, but agreed from the example publish that we no longer have an empty landing page.

**Pull request merged**